unlock-app/src/__tests__/services/walletService.test.js

```diff
@@ -16,6 +15,11 @@ import {
   FAILED_TO_UPDATE_KEY_PRICE,
   FAILED_TO_WITHDRAW_FROM_LOCK,
 } from '../../errors'
+import { POLLING_INTERVAL } from '../../constants'
+
+jest.mock('../../utils/promises')
+const WalletService = require('../../services/walletService').default
```

Comment:
Huuuhuhuh I wish you'd use a regular import here. Please fix this or add a comment about why you need to use `require` here? This is going to be extremely confusing in 2 weeks.